README.md

```diff
@@ -142,6 +142,21 @@ cd Jellyfin.Server/bin/Debug/net8.0 # Change into the build output directory
 
 2. Execute the build output. On Linux, Mac, etc. use `./jellyfin` and on Windows use `jellyfin.exe`.
 
+### Running from GH-Codespaces
+
+Building and running the server from Github Codespaces require you to install the recommended VS-Code extensions as listed in the `.vscode/extensions.json` file. VsCode should ask you to install them each time you open the Server repro on GH-Codespaces.
+
+As Jellyfin will run on a container on a github hosted server, JF needs to handle 3 things differently.
+1. Address Binding.
+Github will put all networked services behind a reverse proxy and will only autodetect it correctly if it will bind to `Ipv4.Any`. Because of this, there is an adapted `BindAllNetworkManager` that will overwrite all set bindings and bind to any IPV4 address.
+**NOTE:** If you want to access the JF instance from outside, like with a WebClient on another PC, remember to set the "ports" in the lower VsCode window to public.
+
+2. FFmpeg installation.
+Because sometimes you need FFMPEG to test certain cases, follow the instructions from the wiki on the dev enviorment:
+https://jellyfin.org/docs/general/installation/linux/#ffmpeg-installation
```

Comment:
Can't we do this as part of the container creation process?

Reply:
Yes absolutly but this is just the basic setup. I plan to create different modules for development and for testing for example. But this is the KISS approach (as it again already takes 2 months to merge this little)